Couple fixes

Rick8484 opened the pull request:
The breakemultiplier value is extremely exaggerated. The train stops less then a second with full load while going top speed. The previous breakemultiplier was 3.

Rick8484 committed:
Update immersiverailroading.cfg

The breakemultiplier value is extremely exaggerated. The train stops less then a second with full load. The previous breakemultiplier was 3.

Rick8484 committed:
Update en_us.lang

Rick8484 committed:
Update EpoxyChain.groovy

Rick8484 committed:
Update en_us.lang

Rick8484 committed:
refresh lead acid texture

These are from trexxet, I believe he forgot.

Rick8484 committed:
Update CeriumChain.groovy

Rick8484 committed:
guh

Rick8484 committed:
Revert "refresh lead acid texture"

This reverts commit 42970f832eeff194d87007e5df7bafa139bd2f6a.

Rick8484 committed:
Make this loop + add better Si recipe

Rick8484 committed:
Update Photomasks.groovy

Rick8484 committed:
forgot to yeet this

Rick8484 committed:
and somebody else forgot to yeet this

Rick8484 committed:
101% sure planetme forgot to remove these

Rick8484 committed:
.

Rick8484 committed:
this too

Rick8484 committed:
Update MonaziteProcessing.groovy

Rick8484 committed:
Update ThoriumChain.groovy

Rick8484 committed:
Imagine forgetting this :skull:

Rick8484 committed:
Update GregTech.groovy

Rick8484 committed:
Merge branch 'master-ceu' into patch-3

trainvoi commented on groovy/postInit/chemistry/inorganic_chemistry/elements/p_block/group14/SiliconChain.groovy:
```diff
@@ -44,58 +44,48 @@ mods.gregtech.electric_blast_furnace.removeByInput(120, [metaitem('dustSiliconDi
+                
+                ADVANCED_ARC_FURNACE.recipeBuilder()
+                        .circuitMeta(1)
+                        .notConsumable(fluid('air') * 10000)
```
make the not consumable 100L of air for every recipe

Rick8484 replied:
done. all of them are 100L

Rick8484 committed:
Update SiliconChain.groovy

trainvoi commented on groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/actinides/ThoriumChain.groovy:
```diff
@@ -24,12 +24,12 @@ BR.recipeBuilder()
+    .inputs(ore('dustSodiumHydroxide'))
+    .fluidInputs(fluid('dissolved_thorium_concentrate') * 2820)
+    .fluidInputs(fluid('tributyl_phosphate_extraction_mixture') * 6400)
+    .chancedOutput(metaitem('dustMonaReHydroxides') * 4, 500, 0)
```
make this 20% instead of 4 * 5% chanced output

Rick8484 replied:
there is no difference

Rick8484 replied:
actually nvm, I cant math

trainvoi commented on groovy/postInit/chemistry/inorganic_chemistry/elements/f_block/actinides/ThoriumChain.groovy:
```diff
@@ -24,12 +24,12 @@ BR.recipeBuilder()
+    .fluidInputs(fluid('tributyl_phosphate_extraction_mixture') * 6400)
+    .chancedOutput(metaitem('dustMonaReHydroxides') * 4, 500, 0)
+    .fluidOutputs(fluid('thorium_extract') * 6400)
+    .duration(160)
```
x4 the duration, so it's 120 to 480

Rick8484 committed:
Update ThoriumChain.groovy

Rick8484 committed:
Update ThoriumChain.groovy

Rick8484 committed:
Update ThoriumChain.groovy

Rick8484 committed:
guh

Rick8484 committed:
Update ChemistryOverhaul.groovy

Rick8484 committed:
Update SusyCore.groovy

Rick8484 committed:
Update PolyvinylChain.groovy

Rick8484 committed:
unless saying otherwise, I wont Multiply  those recipes.

Rick8484 committed:
opsi

Rick8484 committed:
.

Rick8484 committed:
L

Rick8484 committed:
a small buff + heating element

Rick8484 committed:
.

Rick8484 committed:
.

Rick8484 committed:
Update OrganicChemistryMaterials.groovy

Rick8484 committed:
.

Rick8484 committed:
.

Rick8484 committed:
Update Magnets.groovy

Rick8484 committed:
Update BoronChain.groovy

Rick8484 committed:
fix dupe

Rick8484 committed:
pink

Rick8484 committed:
not intended recipe

Rick8484 committed:
.

Rick8484 committed:
Update LanthanumChain.groovy

Rick8484 committed:
.

Rick8484 committed:
.

Rick8484 committed:
.

Rick8484 committed:
guh

bruberu committed:
fix: change bast pr-nd hydroxide recipe to use crystallizers
on planet's order, although it does make sense since it's a precipitation, presumably
